CONTRIBUTING.md

```diff
@@ -66,97 +67,32 @@ clients that need to extract a single description/example.
 
 ## Submitting a pull request
 
-TL;DR: fork, clone, `npm install`, feature branch, commit, push, pull request, check Travis.
+For submitting changes, you can use whatever workflow you're more comfortable with.
+The easiest way is to just edit the page directly on the Github interface.
+Check out the step-by-step instructions (with screenshots) on
+[Github Help](https://help.github.com/articles/editing-files-in-another-user-s-repository/).
 
-Detailed explanation:
+Alternatively, you can do it using the command line:
```

Comment:
I think adding a 1 line command example with each of these points will go a long way. For a newcomer, its extremely easy to just copy-paste the commands and immediately start working on a new command. :smile: